### DISCUSSION TITLE
Snapping to custom angles when shift+rotating in the costume editor

### DISCUSSION THREAD

**Discussion by EightSidedBrick** [ScratchAddons/ScratchAddons#5851] [Ideas]
When I shift+rotate an object in the costume editor, the rotation snaps to 45 degrees. Would be helpful if it was possible to change the angle.

**Comment by Samq64** [comment]
Related: #5814

**Reply by DNin01** [reply]
That issue is for sprite rotation. This discussion is suggesting customizing costume editor shape rotation.
